script/periphery/CreateMerkleLL.s.sol

```diff
@@ -3,20 +3,20 @@ pragma solidity >=0.8.22 <0.9.0;
 
 import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 
-import { ISablierV2LockupLinear } from "../../src/core/interfaces/ISablierV2LockupLinear.sol";
+import { ISablierLockupLinear } from "../../src/core/interfaces/ISablierLockupLinear.sol";
 import { LockupLinear } from "../../src/core/types/DataTypes.sol";
-import { ISablierV2MerkleLL } from "../../src/periphery/interfaces/ISablierV2MerkleLL.sol";
-import { ISablierV2MerkleLockupFactory } from "../../src/periphery/interfaces/ISablierV2MerkleLockupFactory.sol";
+import { ISablierMerkleLL } from "../../src/periphery/interfaces/ISablierMerkleLL.sol";
+import { ISablierMerkleLockupFactory } from "../../src/periphery/interfaces/ISablierMerkleLockupFactory.sol";
 import { MerkleLockup } from "../../src/periphery/types/DataTypes.sol";
 
 import { BaseScript } from "../Base.s.sol";
 
 contract CreateMerkleLL is BaseScript {
     /// @dev Deploy via Forge.
-    function run() public virtual broadcast returns (ISablierV2MerkleLL merkleLL) {
+    function run() public virtual broadcast returns (ISablierMerkleLL merkleLL) {
         // Prepare the constructor parameters.
-        ISablierV2MerkleLockupFactory merkleLockupFactory =
-            ISablierV2MerkleLockupFactory(0xF35aB407CF28012Ba57CAF5ee2f6d6E4420253bc);
+        ISablierMerkleLockupFactory merkleLockupFactory =
+            ISablierMerkleLockupFactory(0xF35aB407CF28012Ba57CAF5ee2f6d6E4420253bc);
```

Comment:
let's add a TODO, to udpate these once deployed new contracts so that we don't forget

Reply:
Should we add TODO in Fork.t.sol as well?